files/image_config/ntp/mgmtvrf_ntpq

```diff
@@ -0,0 +1,5 @@
+ip link set dev lo master mgmt
```

Comment:
what is the purpose for setting lo to master mgmt?

Reply:
ntpq works only when loopback interface is part of mgmt vrf. Hence, loopback is temporarily added to mgmt vrf and moved back after executing the ntpq command.

Reply:
I found this is not intuitive. can you split the PR into two, one for management vrf only, without moving ntpd into management vrf and another one with ntpd move into the vrf. For the first PR, we still let ntp to run under default vrf, I think we are good and is able to approve.